tests/Unit/Evolution/Systems/NewtonianEuler/Test_Primitives.cpp

```diff
@@ -0,0 +1,80 @@
+// Distributed under the MIT License.
+// See LICENSE.txt for details.
+
+#include <memory>
+
+#include "DataStructures/Tensor/Tensor.hpp"
+#include "ErrorHandling/Error.hpp"
+#include "Evolution/Systems/NewtonianEuler/PrimitiveVariables.hpp"
+#include "tests/Unit/Evolution/Systems/NewtonianEuler/TestHelpers.hpp"
+#include "tests/Unit/TestHelpers.hpp"
+
+namespace {
+template <size_t Dim>
+void check_velocity(const DataVector& used_for_size) {
```

Comment:
These should both check operating on doubles.  See, e.g., `tests/Unit/PointwiseFunctions/GeneralRelativity/Test_Christoffel.cpp` for a nice way of doing that.